setup.py

```diff
@@ -97,7 +97,6 @@
     install_requires=[
         "click>=8.1.6,<9",
         "ijson>=3.1.4,<4",
-        "importlib-metadata",
```

Comment:
this is bundled python 3.8+?

Reply:
Yes, in 3.8+, you can now use `importlib.metadata` directly, so we could have done this a while ago

Reply:
Ugh - I guess in 3.8 you can't select by group.
Whyyy Python whyy.